database/migrations/2024_05_16_123152_add_client_id_to_projects_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('projects', function (Blueprint $table) {
+            $table->unsignedBigInteger('client_id')->after('uuid')->nullable();
+        });
+
+        DB::table('projects')->update([
+            'client_id' => DB::table('clients')->where('is_default', true)->value('id'),
```

Comment:
if no default client exists, select first client id.
This way it is a little safer that this migration is successful.